src/main/java/games/strategy/engine/framework/ui/NewGameChooserEntry.java

```diff
@@ -186,4 +171,14 @@ public boolean equals(final Object obj) {
     }
     return this.gameNameAndMapNameProperty.equals(other.gameNameAndMapNameProperty);
   }
+
+
+  @Override
+  public int compareTo(NewGameChooserEntry o) {
+    return getLowerCaseComparable(this).compareTo(getLowerCaseComparable(o));
+  }
+
+  private String getLowerCaseComparable(final NewGameChooserEntry newGameChooserEntry) {
```

Comment:
After moving this method to `NewGameChooserEntry`, it can now be declared static.

Reply:
Or we make use of the current instance we have...